commit 121120c002ada3654b3321667b153041869aab73
Author: Trent Mick <trentm@gmail.com>
Date:   2019-03-22T09:27:15-07:00 (7 months ago)
    
    TRITON-1345 fix sdc-manta conditional inclusion of agent_prebuilt Makefile so can 'make check' on non-SmartOS
    Reviewed by: Tim Foster <tim.foster@joyent.com>
    Approved by: Tim Foster <tim.foster@joyent.com>

diff --git a/Makefile b/Makefile
index 23ca3ca..aad8213 100644
--- a/Makefile
+++ b/Makefile
@@ -171,8 +171,10 @@ publish: release
 CLEAN_FILES += node_modules
 
 include ./deps/eng/tools/mk/Makefile.deps
-include ./deps/eng/tools/mk/Makefile.node_prebuilt.targ
-include ./deps/eng/tools/mk/Makefile.agent_prebuilt.targ
+ifeq ($(shell uname -s),SunOS)
+    include ./deps/eng/tools/mk/Makefile.node_prebuilt.targ
+    include ./deps/eng/tools/mk/Makefile.agent_prebuilt.targ
+endif
 include ./deps/eng/tools/mk/Makefile.targ
 
 MAN_SECTION	:= 1
